docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -206,24 +204,14 @@ Preparing For a Release
       git commit -am "Updated Vagrantfile"
       git push --set-upstream origin release/flocker-${VERSION}
 
-#. Set up Google Cloud Storage and Amazon S3 credentials:
-
-   Creating the Vagrant machine attempts to copy the ``~/.boto`` configuration file from the host machine.
+#. Set up ``AWS Access Key ID`` and ``AWS Secret Access Key`` Amazon S3 credentials:
 
-   Run:
+   Creating the Vagrant machine attempts to copy the ``~/.aws`` configuration directory from the host machine.
+   This means that ``awsutil`` may have correct defaults.
 
    .. prompt:: bash [vagrant@localhost]$
 
-     gsutil ls gs:// s3://
-
-   If the credentials have been set up correctly, you should see ClusterHQ's ``gs://`` and ``s3://`` buckets.
-   If they have not, run:
-
-   .. prompt:: bash [vagrant@localhost]$
-
-      gsutil config
-
-   and set ``aws_access_key_id`` and ``aws_secret_access_key`` in the ``[Credentials]`` section of ``~/.boto`` to allow access to Amazon `S3`_ using `gsutil`_.
+      aws configure
```

Comment:
It seems worth adding the dependency to avoid any extra steps, even simple ones.  The aws CLI is not a particularly heavy dependency, either.